### ISSUE TITLE
Anchors with slashes on Windows are converted to backslashes

### ISSUE THREAD

**Issue by dweiss** [timaschew/link-checker#34]
An anchor link of the form:
```
<a href="target.html#/foo/bar">link</a>
```
causes an incorrect error to be printed:
```
anchor not found from source.html to target.html#\foo\bar
```

Note the backslashes - they're converted probably as part of fs path resolution somewhere, but they really shouldn't be touched. It's the anchor part of the link.
